test/BabelRewirePluginTransformTest.js

```diff
@@ -50,24 +50,33 @@ describe('BabelRewirePluginTest', function() {
 		var input = fs.readFileSync(path.resolve(directory, 'input.js'), 'utf-8');
 		var expected = fs.readFileSync(path.resolve(directory, 'expected.js'), 'utf-8');
 
+		// For non-windows system such as Linux, the test fails because
+		// expected has \r\n, windows line ending, while the output has
+		// just \n. To fix this just replace \r\r with os.EOL.
```

Comment:
I guess this should be

```suggestion
		// just \n. To fix this just replace \r\n with os.EOL.
```

But no big deal

Reply:
Yep, fixed thanks!